tests/nn/data_parallel/test_fsdp.py

```diff
@@ -77,6 +77,49 @@ def get_wrapped_model(group, cuda_first=False, config={}, **model_kwargs) -> Ful
             model = FullyShardedDataParallel(TransformerWithSharedParams(group, **model_kwargs), group, **config).cuda()
         return model
 
+    @classmethod
```

Comment:
assume this is just code movement, no logical changes?

Reply:
yep, just moved it into DistributedTest to be reused in the other file